aya/src/sys/netlink.rs

```diff
@@ -277,7 +277,10 @@ struct Request {
 struct TcRequest {
     header: nlmsghdr,
     tc_info: tcmsg,
-    attrs: [u8; 64],
+    // The buffer for attributes should be sized to hold at least 256 bytes,
+    // based on `CLS_BPF_NAME_LEN = 256` from the kernel:
+    // https://github.com/torvalds/linux/blob/02aee814/net/sched/cls_bpf.c#L28
+    attrs: [u8; 512],
```

Comment:
at least 256, but then why 512?

Reply:
Because we currently use around ~30 bytes of attributes in addition to name.
Rather than picking a "right sized buffer" for the payload (which is of varying length anyway) we use the next largest power of 2.
I've got netlink changes planned soon, so will look at making the buffer size variable then.

Reply:
Can the comment say that please?

Reply:
Sure, will update the comment.

Reply:
@tamird Updated the comment.